common/static/coffee/src/discussion/utils.coffee

```diff
@@ -113,7 +113,7 @@ class @DiscussionUtil
       alertDiv = $("<div class='modal' role='alertdialog' id='discussion-alert' aria-describedby='discussion-alert-message'/>").css("display", "none")
       alertDiv.html(
         "<div class='inner-wrapper discussion-alert-wrapper'>" +
-        "  <button class='close-modal dismiss' aria-hidden='true'><i class='icon fa fa-times'></i></button>" +
+        "  <button class='close-modal dismiss' aria-hidden='true'><span class='icon fa fa-times' aria-hidden='true'></span></button>" +
```

Comment:
As it is in this line of code, you have a button with an icon and nothing beyond the icon that identifies what the button does. If you're a sighted user you see an "X" and you probably recognize it as a close button. But screenreaders only hear "button".

Using `title` on a button will not require `sr` text as a label.

`<button class="close-modal dismiss" title="Close modal">
    <span class="icon fa fa-times" aria-hidden="true"></span>
</button>`